src/background/hypothesis-chrome-extension.js

```diff
@@ -253,15 +273,15 @@ export default function HypothesisChromeExtension({
       // that modify the URL fragment as they load. See commit 3143ca27e05d.
       Object.assign(config, directLinkQuery);
 
-      return sidebar
```

Comment:
Nobody is consuming this promise.

Reply:
This is true. I guess the original reason it was done this way was to make it possible for any callers to tell when `updateTabDocument` had finished whatever work it is going to do.